app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -393,6 +460,12 @@ public void handleRadiusPacket(RADIUS radiusPacket)
                 break;
             case RADIUS.RADIUS_CODE_ACCESS_ACCEPT:
                 log.debug("RADIUS packet: RADIUS_CODE_ACCESS_ACCEPT");
+                if (radiusPktIdentifier == (byte) 0) {
```

Comment:
Try if we can replace this condition with something more clear method like

isRadiusResponseForStatusRequest(identifier)

Same for other such conditions as well. Also, these methods should be put into RadiusOperationalStatusManager class. Try not to pollute AAAManager with logic that can be put into RadiusOperationalStatusManager class